flink-runtime-web/web-dashboard/src/app/pages/job/exceptions/job-exceptions.component.ts

```diff
@@ -61,7 +67,22 @@ export class JobExceptionsComponent implements OnInit {
           this.rootException = 'No Root Exception';
         }
         this.truncated = exceptionHistory.truncated;
-        this.listOfException = exceptionHistory.entries;
+        this.listOfException = exceptionHistory.entries.map(entry => {
+          const values: ExceptionInfoInterface[] = [];
+          values.push(entry);
+          entry.concurrentExceptions.forEach(ex => values.push(ex));
```

Comment:
nit: `values = values.concat(concurrentExceptions).map(markGlobalFailure)`
where
```
function markGlobalFailure(exception: ExceptionInfoInterface) {
    if (exception.taskName == null) {
        exception.taskName = '(global failure)';
    }
    return exception;
}
```

This is all subjective style, but I find it easier to read...